i18n/config.ts

```diff
@@ -0,0 +1,31 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+
+import translationEN from './translations/en-US.json';
+import translationPT from './translations/pt-BR.json';
+
+i18n
+  // pass the i18n instance to react-i18next.
+  .use(initReactI18next)
+  // init i18next
+  // for all options read: https://www.i18next.com/overview/configuration-options
+  .init({
+    // in case window.VEDACCIO_LANGUAGE is undefined,it will fall back to 'en-US'
+    lng: window.VEDACCIO_LANGUAGE,
```

Comment:
Here should be 
```
lng: window?.__VERDACCIO_BASENAME_UI_OPTIONS?.language,
```

Reply:
and then update ...
![Screen Shot 2020-03-03 at 7 47 38 AM](https://user-images.githubusercontent.com/558752/75749936-4ae54780-5d23-11ea-9874-8d1f3cf05c4e.png)

All new config comes from `__UI_OPTIONS`

Reply:
oh yeah...I forgot to add back the English language